### ISSUE TITLE
Priority order should default to zero

### ISSUE THREAD

**Issue by DLWoodruff** [RalfGollmer/ddsip#5]
Presently, to avoid an error message about a missing priority order file, one must have
CPLEXBEGIN
2020 0 *Priority order
CPLEXEND

It would be (slightly) better if this was not required. I.e., it would be better if the value was zero when no value is specified.

**Comment by RalfGollmer** [comment]
I did notice this effect with the order file, too. Suppose it is due to 
a change of CPLEX's default from 12.6.3 to 12.7.0.
I already adapted ddsip to that. such that an order file should 
(hopefully) be asked for just when there is a line
2020 1 *Priority order in the specs.
Noticed this first when testing the new version at the neos server, 
where I got interrupts without any error message in the output (which 
doesn't include the screen output).
Thus I added more error messages to sip.out in order to make it easier 
to pinpoint problems on the neos server.
Today's Github state should not have the problem you noticed.
But I think it will be even better to change to a separate parameter in the specs file, specifying whether a CPLEX order file should be read. I'll do this in the next days.
